### ISSUE TITLE
🐛 [User-reported Bug]

### ISSUE THREAD

**Issue by Turius** [FCP-INDI/C-PAC#1417]
The C-PAC graphical interface to create pipelines doesn't work either on Firefox or Edge.
Specifically, the interface does not allow the user to download the file to run the pipeline locally.
The interface was reached at: https://fcp-indi.github.io/C-PAC_GUI/versions/latest/browser/#/

**Comment by SucyLi** [comment]
Hi, thanks for using C-PAC. 2 questions:

1 Did you edit the default template? The default template can't be downloaded. You should duplicate the default template and edit a new one as shown in image:

![image](https://user-images.githubusercontent.com/34257357/103294210-6363d400-49bf-11eb-8bbe-fa7357f6d65b.png)

2 If you already edited a new template and it still couldn't be downloaded, have you changed any configuration options? Could you please let us know what you changed? I have tried that it can be downloaded if you don't change anything so knowing what options trigger the issue can help us identify the potential bug.

**Comment by Turius** [comment]
Yes I did duplicate it and edited it. I assigned it a custom name and still could not download it. If I try to reach the website now it does not load.
I can not check which options I edited now, but they were mainly in "General" under the number of parallel participants and under "Functional" for distortion correction if I remember correctly.
![Screenshot_2020-12-29 C-PAC](https://user-images.githubusercontent.com/31317716/103297552-d2a9ea00-49f8-11eb-9062-bb771bf2cf03.png)

**Comment by SucyLi** [comment]
Could you try to use Chrome? I edited the number of parallel participants and turned on distortion correction, and I was able to download the config using Chrome.

**Comment by Turius** [comment]
I can download the config using Chrome. Firefox and Edge still not working
